### ISSUE TITLE
Dimension Stacking Issues (1.15.2)

### ISSUE THREAD

**Issue by lividhen** [qouteall/ImmersivePortalsModForForge#77]
Im not 100% sure if this is another mod i have, or if it is immersive portals, but when i fall through or climb up to a different dimension moving vertically becomes a challenge because after a couple blocks i cant fall or go up any more without moving horizontally. normal blocks also half way act like entities, in which i mean i vibrate when standing on top of them and fall through all of them if i hit them hard
. this is all fixed by leaving the dimension another way or dyeing. not sure if this is fixable / solvable because i have 230-ish mods 😅.

[latest.log](https://github.com/qouteall/ImmersivePortalsModForForge/files/4498020/latest.log)

**Comment by qouteall** [comment]
Does this happen with stacked vanilla dimensions?

**Comment by lividhen** [comment]
yes, it does. at one point i only had the nether, overworld, and end stacked and it still did it.

**Comment by qouteall** [comment]
Can you show a video about this?

**Comment by lividhen** [comment]
https://drive.google.com/open?id=1eIhXC3_Dd0IJ-uUZcQ-ptlRiseFIpL8L

**Comment by qouteall** [comment]
Very weird. This could be a mod compatibility issue.

**Comment by qouteall** [comment]
I accidentally reproduced similar issue on multiplayer
